minecraft/src/main/java/org/spicord/velocity/SpicordVelocity.java

```diff
@@ -60,7 +62,16 @@ public SpicordVelocity(ProxyServer proxyServer) {
 
     @Subscribe
     public void onProxyInitialize(ProxyInitializeEvent event) {
-        this.loader.load();
+        final int loadDelay = loader.getConfig().getLoadDelay();
+
+        getLogger().info("Spicord will load in " + loadDelay + " seconds");
+
+        loader.getThreadPool().schedule(() -> {
+            VelocityJDADetector.checkOtherJDA(this);
+            loader.load();
+        }, loadDelay, TimeUnit.SECONDS);
+
+        Fixes.checkLoader(this, false);
```

Comment:
The "Fixes" class is only for Bungee & Bukkit, it will not work at all with Velocity.